hrm_oo.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import numpy as np
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
+                 brady_limit=60, tachy_limit=100):
+        self.extract_vals(file, start_min, end_min, brady_limit, tachy_limit)
```

Comment:
You should consider having all of those input variables get mapped to object attributes if those values might have value / usage after the object is initialized.  Otherwise they are lost of the object is created.